src/statuses.ts

```diff
@@ -0,0 +1,7 @@
+export const statuses: {[code: string]: number | undefined} = {
```

Comment:
Comment?

Reply:
What needs to be commented here?

Reply:
just what is this lookup table for

Reply:
I guess I can just rename `code` to `errorCode` and alias `number` as `HttpStatusCode`?